extensions/ui/org.eclipse.smarthome.ui.paper/package.json

```diff
@@ -12,6 +12,7 @@
     "angular-animate": "1.4.8",
     "angular-aria": "1.4.8",
     "angular-material": "1.0.1",
+    "angular-material-expansion-panel": "^0.7.2",
```

Comment:
Note that for stuff that we package we have to stick to the exact version, only for build&test dependencies like gulp we can go higher.